app/views/candidate_interface/unsubmitted_application_form/show.html.erb

```diff
@@ -34,203 +34,9 @@
-            path: @application_form_presenter.becoming_a_teacher_path,
-            custom_status: @application_form_presenter.becoming_a_teacher_review_pending? ? 'Review' : nil,
-            custom_color: @application_form_presenter.becoming_a_teacher_review_pending? ? 'pink' : nil,
-          )) %>
-        </li>
-        <li class="app-task-list__item govuk-hint">
-          <%= render(TaskListItemComponent.new(
-            text: t('page_titles.subject_knowledge'),
-            completed: @application_form_presenter.subject_knowledge_completed?,
-            path: @application_form_presenter.subject_knowledge_valid? ? candidate_interface_subject_knowledge_show_path : candidate_interface_edit_subject_knowledge_path,
-            custom_status: @application_form_presenter.subject_knowledge_review_pending? ? 'Review' : nil,
-            custom_color: @application_form_presenter.subject_knowledge_review_pending? ? 'pink' : nil,
-          )) %>
-        </li>
-      </ol>
-    </section>
-
-    <section class="govuk-!-margin-bottom-8">
-      <h2 class="govuk-heading-m govuk-!-font-size-27"><%= t('section_groups.adjustments') %></h2>
-
-      <ol class="app-task-list">
-        <li class="app-task-list__item">
-          <%= render(TaskListItemComponent.new(
-            text: t('page_titles.training_with_a_disability'),
-            completed: @application_form_presenter.training_with_a_disability_completed?,
-            path: @application_form_presenter.training_with_a_disability_valid? ? candidate_interface_training_with_a_disability_show_path : candidate_interface_new_training_with_a_disability_path,
-          )) %>
-        </li>
-        <li class="app-task-list__item govuk-hint">
-          <%= render(TaskListItemComponent.new(
-            text: t('page_titles.interview_preferences'),
-            completed: @application_form_presenter.interview_preferences_completed?,
-            path: @application_form_presenter.interview_preferences_valid? ? candidate_interface_interview_preferences_show_path : candidate_interface_new_interview_preferences_path,
-          )) %>
-        </li>
-      </ol>
-    </section>
-
-    <section class="govuk-!-margin-bottom-8">
-      <h2 class="govuk-heading-m govuk-!-font-size-27"><%= t('section_groups.safeguarding') %></h2>
-
-      <ol class="app-task-list">
-        <li class="app-task-list__item">
-          <%= render(TaskListItemComponent.new(
-            text: t('page_titles.suitability_to_work_with_children'),
-            completed: @application_form_presenter.safeguarding_completed?,
-            path: @application_form_presenter.safeguarding_valid? ? candidate_interface_review_safeguarding_path : candidate_interface_new_safeguarding_path,
-          )) %>
-        </li>
-      </ol>
-    </section>
+    <% cache @application_cache_key do %>
+      <%= render 'task_list', application_form: @application_form, application_form_presenter: @application_form_presenter %>
+    <% end %>
```

Comment:
Moving the task list into a partial makes it easier to see that this part of the template is cached.

Reply:
Should we set an expiry time on the cache? eg if the template itself changes then potentially the user could never see UI changes

Reply:
What happens to the cache on re-deploys? Does it get automatically cleared?

Reply:
in-memory i assume it would, but maybe not in redis?

Reply:
Yep, can set maybe 5 minutes on this for now? Automatically clearing these cache entries on deploy is something we'd have to set up.